tests/Feature/Sensors/QueuedJobSensorTest.php

```diff
@@ -110,6 +110,7 @@
 
     $response->assertOk();
     $ingest->assertWrittenTimes(1);
+    $ingest->assertLatestWrite('queued-job:*', []);
```

Comment:
This test really needs this assertion to truly check what it is testing.

You can tell cause I had to remove the job that was dispatching to the queue 😄